discovery-infra/start_discovery.py

```diff
@@ -291,6 +303,84 @@ def validate_dns(client, cluster_id):
         raise e
 
 
+def configure_terraform_workers_nodes(tf_folder, num_worker_nodes):
+    tfvars = utils.get_tfvars(tf_folder)
+    num_workers = tfvars['worker_count']
+    num_workers = num_workers + num_worker_nodes
+    tfvars['worker_count'] = num_workers
+    master_end_ip = tfvars['libvirt_master_ips'][-1]
+    workers_ip_list = tfvars['libvirt_worker_ips']
+    if not workers_ip_list:
+        worker_starting_ip = ipaddress.ip_address(master_end_ip)
+    else:
+        worker_starting_ip = ipaddress.ip_address(tfvars['libvirt_worker_ips'][0])
+    worker_ips_list = workers_ip_list + _create_ip_address_list(num_worker_nodes, worker_starting_ip + 1)
+    tfvars['libvirt_worker_ips'] = worker_ips_list
+    utils.set_tfvars(tf_folder, tfvars)
+
+
+def configure_terraform_api_dns(tf_folder, api_vip_ip, api_vip_dnsname):
+    main_str = utils.get_tf_main(tf_folder)
```

Comment:
can't we have dns as part of tfvars.json?

Reply:
and just do the same as with all other params?

Reply:
aw we agreed, changing dns by changing the api-vip and the cluster-name in the tfvars